lib/cdo/pardot.rb

```diff
@@ -315,7 +315,7 @@ def self.submit_prospect_batch(prospects, config)
     unless malformed_emails.empty?
       PEGASUS_DB[:contact_rollups].
         where("email in ?", malformed_emails).
-        update(email_malformed: true)
+        update(email_malformed: true, pardot_sync_at: DateTime.now)
```

Comment:
Change: also update the `pardot_sync_at` field to current time (from NULL) if email is rejected as invalid. This doesn't change any other behavior -- all the code will react the same way in current and future runs with or without this change. But it makes it easier to understand and validate the data. As I was double-checking the Pardot sync process, the absence of `pardot_sync_at` for these records kept being confusing and looking like a problem. Now the absence of this field means the record something new that needs to be synced, and the presence of it means the record is (successfully or unsuccessfully) synced and needs no further action.